JourneysSquad/JourneysSquad/View/BillRowView.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+/// A view displaying one bill.
+struct BillRowView: View {
+
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                let description = "Description"
+                if description.count > 20 {
+                    Text(description.prefix(20) + "...")
+                        .font(.system(size: 14))
+                } else {
+                    Text(description)
+                        .font(.system(size: 14))
+                }
+
+                let participants = ["Participant 1", "Participant 2", "Participant 3"]
```

Comment:
```suggestion
                let participants = [
                "Participant 1", 
                "Participant 2",
                 "Participant 3",
                 ]
```
nit: 
It's easer to read.

Reply:
You right. 
In the current implementation with temporary data, I displayed two participants. I think describing them in one line doesn't make the code harder to read.